src/pint/models/astrometry.py

```diff
@@ -69,6 +63,22 @@ def ssb_to_psb_xyz_ECL(self, epoch=None):
         # TODO: would it be better for this to return a 6-vector (pos, vel)?
         return self.coords_as_ECL(epoch=epoch).cartesian.xyz.transpose()
 
+    def sun_angle(self, toas):
+        """Compute the pulsar-Earth-Sun angle.
```

Comment:
Technically this is the pulsar-observatory-Sun angle. Some of our observatories are not on the Earth.
Also, maybe add a clarification to the documentation saying that this is "the angle from the center of the Sun to the pulsar direction, as seen from the observatory"